test/package.json

```diff
@@ -6,14 +6,14 @@
   "scripts": {
     "debug": "node --preserve-symlinks test.js test --framework karma --log-level debug",
     "test": "node --preserve-symlinks test.js test --framework karma",
-    "coverage": "node --preserve-symlinks test.js test --framework karma --config ./node_modules/@bedrock/karma/lib/coverageConfig.js",
-    "coverage-ci": "node --preserve-symlinks test.js test --framework karma --config ./node_modules/@bedrock/karma/lib/coverageConfig.js"
+    "coverage": "cross-env NODE_ENV=test c8 npm run test",
+    "coverage-ci": "cross-env NODE_ENV=test c8 --reporter=lcovonly --reporter=text-summary --reporter=text npm run test"
```

Comment:
@davidlehn, can you look at the changes in this file to make sure it looks good to you?

Reply:
This needs to use our karma coverage support, c8 is for node.

Reply:
Fixed here 4c5a720aff7f923d11e7a4ebe480e0f73b6abc55